testing.sagebrush.master/Stepwat_Inputs/Input/bmassflags.in

```diff
@@ -53,6 +53,8 @@
 #       is reported immediately to the right of the group biomass.
 #       summary: average for the given year. Can only be
 #       output if GrpBmass is specified.
+# Grpwf == wild fires count across all the iterations.
+# Grppf == prescribed fires count across all the iterations.
```

Comment:
`# Grppf == number of prescribed fires that occured across all iterations in each year`

Reply:
Commit 589665a implements this.